templates/network/openvpn_report.sh

```diff
@@ -1,21 +1,11 @@
 #!/bin/bash
 
-function report {
-  while read -r channel; do
-    export gitter_channel="$channel"
-    treehouses feedback "$(treehouses openvpn | tail -n 1 | tr "\n" "#" | sed 's/#/\\n/g')vpn"
-  done < /etc/openvpn_report_channels.txt
-  echo "report"
-}
-
-
-
 prevStatus=$(cat /tmp/prev_openvpn_status 2>/dev/null || echo "offline")
 if ping -I tun0 -q -c 3 -W 1 1.1.1.1 >/dev/null; then
   echo "vpn is online, was $prevStatus"
   if [ "$prevStatus" = "offline" ]; then
     echo "online" > /tmp/prev_openvpn_status
-    report
+    treehouses openvpn notice now
```

Comment:
I don't think `treehouses openvpn notice now` is a valid command.

Reply:
yeah. it was a late night. I think we might have to close this one. I'll leave it open just in case we decide to change it.